packages/@react-spectrum/table/src/table.css

```diff
@@ -27,11 +27,18 @@
   height: 100%;
 }
 
+.react-spectrum-Table-cell--alignStart {
+  text-align: start;
```

Comment:
adding text-align so we handle alignment if user passes in spans as the cell content